test/blackbox-tests/test-cases/melange/reason.t/lib/x.re

```diff
@@ -0,0 +1,3 @@
+let buy_it = "buy " ++ Y.it
+
+let () = print_endline(buy_it)
```

Comment:
We don't actually depend on reason for our tests (too unreliable of a dependency) and make instead use a fake refmt binary. We should use that binary for these tests as well.

Reply:
After further thinking, I don't believe there's a need to test Reason for the `melange` mode, as it should not be affected.